stories/Radio.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { Radio as Component } from "../confetti-ds/src";
+
+export default {
+  title: "Radio/Radio",
+  component: Component,
+  argTypes: {
+    value: {
+      control: {
+        type: "select",
+        options: ["string", "number", "bool"]
+      }
+    }
+  }
+};
+
+export const Radio = (args) => (
+  <fieldset>
+    <Component
+      id="demo-radio-1"
+      name="demo-radio"
+      label="radio 1"
+      value="bool"
```

Comment:
you are not passing the `args` as props to the components, so the controls from storybook aren't changing the components. please look at the Button example to see what I mean and add the args here.

Reply:
thank you, fixed